Makefile

```diff
@@ -126,6 +132,12 @@ push:
 		$(DOCKER) push $(REGISTRY)/$(TARGET):latest; \
 	fi
 
+	if [ "$(GIT_BRANCH)" = "master" ]; \
```

Comment:
This can still be `master` for a pull request if the user is using their own `master` branch even though it wouldn't be advised. I suggest putting this check within the `TRAVIS_PULL_REQUEST` check so that it is only run for official travis builds from `master`.